.github/instructions/README.md

```diff
@@ -0,0 +1,108 @@
+# 🚀 Terraform AzureRM Provider - Development Guidelines

```

Comment:
Since the prompt is a user-interactive file, do we need to add a document to inform users how to use it?